edison/static/js/policy.controller.js

```diff
@@ -0,0 +1,150 @@
+
+import { Policy } from './policy.model.js'
+
+var policy = new Policy();
+
+
+function createInitElement(type, classSet = '', id = '', style = '') {
+
+    var element = document.createElement(type);
+
+    if (classSet != '') {
+        element.setAttribute("class", classSet);
+    }
+    if (id != '') {
+        element.setAttribute("id", id);
+    }
+
+    if (style != '') {
+        element.setAttribute("style", style);
+    }
+
+    return element;
+}
+
+function setCondistion() {
+
+
+    document.getElementById('condition').addEventListener('click', function () {
+
+        var conditionValue = document.getElementById('condition').value;
+        var value = document.getElementById('showValue');
+        var from = document.getElementById('showFrom');
+        var to = document.getElementById('showTo');
+        var lableValue = document.getElementById('lableValue');
+
+        if (conditionValue == '=') {
```

Comment:
This is probably not the best way to handle with the different values of conditionValue.
My suggestion is to create a new class, named "PolicyElementsModifier" for an instance, that contains a dictionary which key is "conditionValue" and value is an array that contains the configurations for the elements.
Inside the constructor you can fill the dictionary with the different conditions.

```
export class PolicyElementsModifier {
  // # is the way to declare private fields.
  #configByCondition;

  constructor() {
    this.#configByCondition = {
      '=': [ 'none', 'none', 'Equal', 'inline' ],
      ...
    }
  }

  modify(condition, from, to, labelValue, value) {
    if (this.#configByCondition.hasOwnProperty(condition)) {
      config = this.#configByCondition[condition];
      from.style.display = config[0];
      to.style.display = config[1];
      ...
    }
  }
}
```
Maybe there's a better way than my suggestion, but it's a progress towards a more maintainable code.
Now, when you want to add an additional condition, you just need to add it to the dictionary.

Reply:
OK, :) done